SPECS/gsettings-desktop-schemas/gsettings-desktop-schemas.spec

```diff
@@ -6,7 +6,6 @@ Name:           gsettings-desktop-schemas
 Version:        3.36.1
```

Comment:
Since it's a migration from `SPECS-EXTENDED` we should also clean it up a little. Running a linter should do it, I'd just check if it didn't clean-up "too much" (it still tends to enclose parameterized macros in curly brackets for instance, which is incorrect).